File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        stack=[]

        for i in logs:
            if i !="./":
                if i!="../":
                    stack.append(i)
                else:
                    try:
                        stack.pop()
                    except:
                        pass
        c=0
        while stack != []:
            stack.pop()
            c=c+1
            
        return c
        

File: 1598-crawler-log-folder/1598-crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        ArrayList<String> stack = new ArrayList<>();
        for (String item : logs) {
            if (!item.equals("./")) {
                if (!item.equals("../"))
                    stack.add(item);
                else {
                    try {
                        stack.remove((stack.size() - 1));
                    } catch (Exception e) {
                        // Block of code to handle errors
                    }
                }
            }
        }

        int c = 0;
        while (stack.size() != 0) {
            stack.remove((stack.size() - 1));
            c++;
        }
        return c;
    }
}